docs/operator-manual/declarative-setup.md

```diff
@@ -150,14 +161,14 @@ spec:
 !!!note
     Some Git hosters - notably GitLab and possibly on-premise GitLab instances as well - require you to
     specify the `.git` suffix in the repository URL, otherwise they will send a HTTP 301 redirect to the
-    repository URL suffixed with `.git`. ArgoCD will **not** follow these redirects, so you have to
-    adapt your repository URL to be suffixed with `.git`.
+    repository URL suffixed with `.git`. Argo CD will **not** follow these redirects, so you have to
+    adjust your repository URL to be suffixed with `.git`.
 
-Repository credentials are stored in secret. Use following steps to configure a repo:
+Repository credentials are stored in the `argocd-secret` secret. Use following steps to configure a repo:
```

Comment:
Repository credentials are not defined in the `argocd-secret` secret, but are stored in unique secrets, which in turn are referenced from the repository's configuration in the `argocd-cm` ConfigMap. For example, consider the following configuration in `argocd-cm`:

```yaml
repositories: |
    - url: https://github.com/argoproj/argo-cd
      passwordSecret:
        name: my-secret
        key: password
      usernameSecret:
        name: my-secret
        key: username
```

would define to read the required credentials from the fields `username` and `password` stored in a secret named `my-secret`.

Reply:
Hmm, indeed, I half claim that fixing this involves significantly more effort, because an entire row should be added to a table (the new narrower table).

I'll try to figure out what it should say...

Reply:
As noted on Slack, I really look forward to the repositories item being composed of n unique config objects instead of a single thing. That's what brought me to this document in the first place.